### DISCUSSION TITLE
Docker. Synology NAS

### DISCUSSION THREAD

**Discussion by bolshukhin** [Tzahi12345/YoutubeDL-Material#514] [Q&A]
Hello. It's my first time of use Youtube-dl. How I can change default folder to Downloads? And how I can change audio formats from .mp3 to flac? Thanks 

Device based on DSM 7.0.1-42218 Update 2

**Comment by GlassedSilver** [comment]
Not sure what you hope to achieve by downloading flac, if you're actually targeting sites compatible with youtube-dl that offer FLAC then best audio SHOULD catch that (detault setting).

As for sites like YouTube, YouTube Music and such, they don't offer FLAC. If you converted them to FLAC you would only change the container and gain no quality.

As for changing the path, you can do that in the settings in the app itself and in the container settings to map the folders to any host path of your choice. :)
Don't ask me how to do it on Synology, I don't run such a system. :)
Google Search might help a lot there, but I believe it shouldn't be too hard to figure it out in Synology's WebUI on your own. :)

Make sure you are on the nightly branch, it's much more feature-rich!

**Comment by amuilyas** [comment]
I run Synology but deploy YTDL using Portainer.  If you are using the Syno Docker program then you edit directory in the volume tab.  Here you can point the audio, video and sub downloads to your specific folders. You need to make sure the Docker user has R/W permissions to the new folders

![image](https://user-images.githubusercontent.com/77816893/219206834-67da345b-4d96-4ba8-a75c-9ce36855008e.png)
